scripts/README.md

```diff
@@ -7,3 +7,24 @@ This also contains scripts for automatically generating schema files associated
 
 - [biomedical](biomedical) is the subdirectory containing scripts to format data for Biomedical
   Data Commons (BMDC).
+
+## Adding a dataset for auto-refresh to DC
+
+Prerequisite: Make sure you have verified the generated MCF before working on auto-fresh MCF. These are documented in [Summer 2020 Intern Data Import Workflow](go/dc-summer2020#2020-intern-workflow).
+- Auto script CSV & MCF 
+  - Add manifest.json under the subdirectory and tag the PR following [import-automation README](../import-automation/README.md).
+  - After submitting PR, check that a [Cloud Scheduler job](https://pantheon.corp.google.com/cloudscheduler?project=google.com:datcom-data&folder=&organizationId=) for the import is created. 
+  - Either wait till the job gets scheduled or manually clicks [RUN NOW](https://screenshot.googleplex.com/5yDXFYh25bZ824d). Upon success,  you shall see data generated under the corresponding [directory](https://pantheon.corp.google.com/storage/browser/datcom-prod-imports/scripts?project=datcom-204919&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=&forceOnObjectsSortingFiltering=false).
+  - [Import Progress Dashboard](https://dashboard-frontend-dot-datcom-data.uc.r.appspot.com/) can be helpful. Click the down arrow at the left side for more information. 
+- Add the import to side cache
+  - Add import to side_cache_mcfs.textproto. 
+    - Set perform_automated_mcf_generation for converting CSV & TMCF to MCF.
+    - Set download_config which tells source CSV & TMCF on bigstore and destination path on CNS.
+    - Set resolution_info. Specify KG_MCF option when need the StatVar nodes.
```

Comment:
I wonder if either the whole README should live in g3, or at least this part onwards... since it will have reference to g3 manifest, controller_import_checker, etc.

Reply:
Good questions. I was considering both G3 & github and picked Gibhub since this is where things get started. 

However, if this parts move to G3, r u thinking about google3/datacommons/import/mcf/README.md?
And this readme would still refer to a g3 side_cache readme?

Reply:
Assuming in the near future googlers only do this, something there or under company-doc (https://g3doc.corp.google.com/company/teams/datacommons/index.md?cl=head) might make sense.

That can be the canonical reference doc.  Since we start out with an inaccessible go-link :), perhaps this could entirely move there?